src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -16,8 +24,39 @@ public LineService(LineRepository lineRepository) {
         this.lineRepository = lineRepository;
     }
 
-    public LineResponse saveLine(LineRequest request) {
-        Line persistLine = lineRepository.save(request.toLine());
+    public LineResponse saveLine(LineRequest request) throws DuplicateEntityExistsException{
```

Comment:
```suggestion
    public LineResponse saveLine(LineRequest request) {
```

throws를 사용해서 컨트롤러 밖으로 예외를 전달할 필요가 있을까요?
현재 메서드 내부에서 예외를 던져도 컨트롤러 어드바이스를 추가했기 때문에 예외 처리 가능합니다.

Reply:
엇 제가 `@ControllerAdvice`의 용법을 제대로 파악하지 못하고 사용하였네요. 감사합니다.